src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.CrossGen.targets

```diff
@@ -198,44 +197,12 @@ Copyright (c) .NET Foundation. All rights reserved.
       <_CrossProjFileDir>$([System.IO.Path]::Combine($(ComposeWorkingDir),"Optimize"))</_CrossProjFileDir>
       <_NetCoreRefDir>$([System.IO.Path]::Combine($(_CrossProjFileDir), "netcoreapp"))</_NetCoreRefDir>              <!-- flat netcore app assemblies-->
     </PropertyGroup>
-  </Target>
-
-  <!--
-    ============================================================
-                                        _GenerateCrossgenProj
-    Generates the project used to restore netcoreapp corresponding to RuntimeFrameworkVersion
-    ============================================================
-    -->
-  <Target Name="_GenerateCrossgenProj"
-          DependsOnTargets="_GetCrossgenProps;"
-         Condition="$(SkipOptimization) != 'true' ">
-    <!-- Create a Project to Restore Crossgen -->
+     <MakeDir  Directories="$(_CrossProjFileDir)"/>
     <PropertyGroup>
-      <CrossgenProjContent>
-        <![CDATA[
-<Project Sdk="Microsoft.NET.Sdk">
-  <PropertyGroup>
-      <TargetFramework>$(_TFM)</TargetFramework>
-      <RuntimeIdentifier>$(RuntimeIdentifier)</RuntimeIdentifier>
-      <RuntimeFrameworkVersion>$(RuntimeFrameworkVersion)</RuntimeFrameworkVersion>
-  </PropertyGroup>
-</Project>        
-
-]]>
-      </CrossgenProjContent>
+      <_CrossProjAssetsFile>$([System.IO.Path]::Combine($(_CrossProjFileDir),  project.assets.json))</_CrossProjAssetsFile>
```

Comment:
Why did `obj` get removed from this path?  Shouldn't the assets file go into the `obj` folder?